Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -76,8 +76,12 @@ public void setWordCount(String url, int wordCount) {
 
   public void insertScriptedData(String url) {
     SQLiteDatabase dataBase = getWritableDatabase();
-    dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
-    dataBase.close();
+    try {
+      dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
+      dataBase.close();
+    } catch (Exception exception) {
+      dataBase.close();
+    }
```

Comment:
Yup. The problem in here is that the code is not handling the exception so we have a empty block. Handle it in any way so we don't skip the exception.